bin/improver-regrid

```diff
@@ -105,10 +100,15 @@ def main():
             extrapolation_mode=args.extrapolation_mode)
 
     source_on_target = source_data.regrid(target_grid, regridder)
-    target_grid_attributes = {k: v for (k, v) in target_grid.attributes.items()
-                              if 'mosg__grid' in k}
-    amend_metadata(source_on_target, revised_attributes=target_grid_attributes)
-    delete_attributes(source_on_target, ['title'])
+    if args.json_file:
+        metadata_dict = load_json(args.json_file)
+        amend_metadata(source_on_target, **metadata_dict)
+    else:
+        target_grid_attributes = (
+            {k: v for (k, v) in target_grid.attributes.items()
+             if 'mosg' in k or 'institution' in k})
+        amend_metadata(source_on_target, attributes=target_grid_attributes)
+        delete_attributes(source_on_target, ['title'])
```

Comment:
Why do we want to delete the "title" as default.

Reply:
This was already in the code before this PR, but I'm not sure we should be doing by default anymore as users can edit or delete it through the json file now.

Reply:
I've removed this, as this can now be done using the json file.